package.json

```diff
@@ -14,8 +14,10 @@
     "test": "echo \"No test specified\"",
     "release": "np --no-2fa",
     "start": "ts-node ./examples/index.ts",
-    "lint": "eslint --max-warnings 0 .",
-    "format": "prettier --config .prettierrc './**/*.ts' --write",
+    "lintCI": "eslint --max-warnings 0 .",
+    "lint": "eslint --fix .",
+    "format": "prettier --config .prettierrc -l './**/*.ts' --write",
```

Comment:
Remove -l? Not sure if it makes a difference

Reply:
Another nit, we usually call it `fmt` from what I have seen

Reply:
list just makes the information into a more digestable format: https://prettier.io/docs/en/cli.html